dotCMS/src/main/java/com/dotcms/rest/api/v1/container/ContainerResourceHelper.java

```diff
@@ -70,6 +76,20 @@ public void setContainerLanguage(final Container container, final HttpServletReq
                 }
             }
         }
-    } // setContainerLanguage.
+    }
+
+    public ContainerWithStructuresView toResponseEntityContainerWithStructuresView(final Container container, final List<ContainerStructure> containerStructures) {
```

Comment:
Please rename this class to `toResponseEntityContainerWithContentTypesView`, as `Structure` was the old term used to refer to what we now call Content Type.

Reply:
this has been considered and code updated accordingly